cmd/status/cmdstatus.go

```diff
@@ -123,7 +122,7 @@ func (r *StatusRunner) runE(cmd *cobra.Command, args []string) error {
 		ErrOut: cmd.ErrOrStderr(),
 	})
 	if err != nil {
-		return errors.WrapPrefix(err, "error creating printer", 1)
+		return fmt.Errorf("error creating printer: %v", err)
```

Comment:
We should probably use `%w` instead of `%v` here so we have the option of unwrapping the error. Same goes for the other changes.

Reply:
Then I suppose it needs to be done consistently across the whole codebase? There are places where errors are not wrapped at all and places that use `%v`. Are these particular spots I'm touching in this PR special in some way that unwrapping must be possible? Just trying to understand.

Reply:
I think we should just use `%w` to wrap the error whenever we use this pattern. I'm sure it is not used consistently across the codebase, but it seems like a good practice.

Reply:
@mortent I agree. Done!